pipelines/jobs/configurations/jdk11u_release.groovy

```diff
@@ -0,0 +1,15 @@
+targetConfigurations = [
```

Comment:
The whitespace formatting is different than the other files.  I don't mind which one but we should probably be consistent :-)